### PR TITLE
Map CSS fonts tests to web-features

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Review by jcscottiii — APPROVED** [review]

File: css/css-fonts/WEB_FEATURES.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - font-palette.html
+  - font-palette-*
+  - palette-values-rule-*
+- name: font-synthesis
+  files:
+  - font-synthesis-*
+- name: font-variant-alternates
+  files:
+  - alternates-order.html
+  - font-variant-alternates-*
```
Comment (jfkthame) [inline css/css-fonts/WEB_FEATURES.yml:13]: Is there any particular reason for which "features" are covered here and which aren't? It seems like there are a bunch more of these tests that could be mapped to a CSS feature; e.g. there are tests in this directory for `font-size-adjust`, `font-weight`, `font-stretch`, other subproperties of `font-variant-*`, and more. So offhand this looks quite incomplete?

**Review by drott — APPROVED** [review]
LGTM, seems to provide good coverage of the listed features. What's the plan on maintaining this going forward?

File: css/css-fonts/WEB_FEATURES.yml (hunk 1 above)
Comment (foolip) [reply css/css-fonts/WEB_FEATURES.yml:13]: These are the font-related ones that are in https://github.com/web-platform-dx/web-features so far, and it is incomplete. `font-size-adjust` is in review at https://github.com/web-platform-dx/web-features/pull/748 and I need to come back to it. In https://github.com/web-platform-dx/web-features/issues/666 we're tracking making web-features more complete with various burndown lists. Feel free to comment there, file issues, or send PRs if there are specific features you'd like to prioritize.

**Comment by foolip** [comment]
> LGTM, seems to provide good coverage of the listed features. What's the plan on maintaining this going forward?

Overall coverage is minuscule right now, but my long term hope is that we will cover all of WPT, and triage all tests into either a feature, or into a "it's complicated" category which we don't think is worth linking to something in web-features. That could be certain crashtests, tests for a cross-cutting issue affecting many features but none more than others, etc.

I think we need to expand coverage a lot before it starts to be attractive for anything to keep coverage high, but at some point I hope that it will be deemed valuable and people might add tests in regular reviews.

It's all set up as separate files to not be intrusive to anyone right now, so that if we need to change something, it's not a change to the tests themselves, which always has the risk of accidentally affecting the test.

**Pull request merged**